src/components/UserMenu.vue

```diff
@@ -5,7 +5,7 @@
       <span class="oc-topbar-personal-label">{{ userDisplayName }}</span>
     </oc-button>
     <oc-drop toggle="#_userMenuButton" mode="click" :options="{pos:'bottom-right'}" class="uk-width-large" ref="menu">
-      <div class="uk-card-body uk-flex uk-flex-middle uk-flex-column">
+      <div class="uk-card-body uk-flex uk-flex-middle uk-flex-column" id="profileInfoContainer">
```

Comment:
I think something like `account-info-container` would be better to keep it in sync with other IDs e. g. https://github.com/owncloud/phoenix/blob/4fd86f4f0f339726d88ef43c8c7872ad8534cdc5/apps/files/src/components/FileList.vue#L27

Reply:
thankyou

Reply:
Thank you too!